app/services/user.ts

```diff
@@ -141,6 +145,9 @@ export class UserService extends PersistentStatefulService<IUserServiceState> {
 
     fetch(request)
       .then(res => {
+        const service = getPlatformService(this.state.auth.platform.type);
+        this.login(service, this.state.auth);
+        this.refreshUserInfo();
         return res.text();
       })
       .then(valid => {
```

Comment:
Don't think it works correctly. this.LOGOUT() will be never called because we didn't use 'handleResponse' method here. It ofter returns '401' for me